cli/azd/pkg/pipeline/testdata/Test_promptForCiFiles-no_files_-_github_selected_-_App_host_-_fed_Cred.snap

```diff
@@ -22,7 +22,7 @@ jobs:
       AZURE_TENANT_ID: ${{ vars.AZURE_TENANT_ID }}
       AZURE_SUBSCRIPTION_ID: ${{ vars.AZURE_SUBSCRIPTION_ID }}
       AZURE_ENV_NAME: ${{ vars.AZURE_ENV_NAME }}
-      AZURE_LOCATION: ${{ vars.AZURE_LOCATION }}
+      AZURE_LOCATION: ${{ vars.AZURE_LOCATION }}      
```

Comment:
nit: Remove whitespace.
```suggestion
      AZURE_LOCATION: ${{ vars.AZURE_LOCATION }}
```

Reply:
This is coming from a go template. I will update the template and re-run the test.
Good catch!